docs/pim/symbol_attribute_type.md

```diff
@@ -0,0 +1,190 @@
+---
+description: Create a symbol attribute type that enables for the efficient representation of string-based values while enforcing their format in product specifications.
+---
+
+# Symbol attribute type
+
+In product specifications, the symbol attribute type enables the efficient representation of string-based data and enforces their format.
```

Comment:
I'd expand this section with a sentence or two about possible use cases.

Something like:
You can use this attribute to store standarized identifiers (like EAN or ISBN) of your products in the Product Information Management system.